lib/FundingProgrammes.php

```diff
@@ -10,10 +10,11 @@
 
 class FundingProgrammeTransformer extends TransformerAbstract
 {
-    public function __construct($locale, $isSingle = false)
+    public function __construct($locale, $isSingle = false, $showAllProgrammes = false)
```

Comment:
I'd be inclined to name this flag `$includeImages` or something that defined the effect, but otherwise 👍